src/ui/MessageSearchFileItem/utils.ts

```diff
@@ -22,15 +22,15 @@ export function getCreatedAt({ createdAt, locale, stringSet }: GetCreatedAtProps
     return '';
   }
   if (isToday(createdAt)) {
-    return format(createdAt, 'p', optionalParam);
+    return format(createdAt, stringSet?.DATE_FORMAT__LAST_MESSAGE_CREATED_AT__TODAY ?? 'p', optionalParam);
```

Comment:
It seems `stringSet` prop shouldn't be an optional type based on the type def in `LocalizationProvider`. Could you check this again?

Reply:
I applied the review, pls check it again!